docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+spectral and delay events and (2) how average signal frequencies and amplitudes
+in each spectral dimension are determined. We outline the procedures for
+designing and creating **TransitionQuery** and **MixingQuery** for single- and
+multi-spin transitions and how to use them to select the transition pathways
+with the desired frequency and amplitudes in each **SpectralDimension** of your
+custom method object. In multi-dimensional spectra, we illustrate how the
+desired frequency correlation can sometimes be achieved by using an appropriate
+affine transformation. We also examine how changing the frequency contributions
+in **SpectralEvent** of **DelayEvent** objects can be used to obtain the desired
+frequency and amplitude behavior. The ability to select :ref:`frequency
+contributions<freq_contrib_api>` can often reduce the number of events needed in
+the design of your custom Method object.
+
+Theoretical Background
+----------------------
+
+Before giving details on how to create a custom Method object, we review a
+few key concepts about spin transitions and *transition symmetry functions*
+
+The number of quantized energy eigenstates for :math:`N` coupled nuclei is
+
+.. math::
+
+    \Upsilon_{\left\{ I_1, I_2, \ldots, I_N \right\}} = \prod_{u=1}^N (2 I_u+1),
+
+where :math:`I_u` is the total spin angular momentum of the :math:`u\text
+{th}` nucleus and the system of coupled nuclei under consideration is
+represented with the notation
+:math:`\left\{ I_1, I_2, \ldots, I_N \right\}`. The transition from quantized
+energy level :math:`E_i` to :math:`E_j` is one of
+
+.. math::
+
+    \mathcal{N}_{\left\{ I_1, I_2, \ldots, I_N \right\}} = \frac{\Upsilon_{\left\{ I_1, I_2, \ldots, I_N \right\}}!}{(\Upsilon_{\left\{ I_1, I_2, \ldots, I_N \right\}}-2)!}
+
+possible transitions between the :math:`\Upsilon` levels.   Here we
+count :math:`i  \rightarrow  j` and :math:`j  \rightarrow  i` as different
+transitions.  For example, a single spin with angular momentum :math:`I=3/2`,
+indicated by :math:`\left\{ I \right\} = \left\{ \tfrac{3}{2} \right\}`, will
+have :math:`\Upsilon_{\left\{ 3/2 \right\}} = 2I+1 = 4` energy levels
+and :math:`\mathcal{N}_{\left\{ 3/2 \right\}} = 2I(2I+1) = 12` possible NMR
+transitions.   A two spin system, :math:`\left\{ I, S \right\} = \left\{ \tfrac
+{1}{2}, \tfrac{1}{2} \right\}`, will have
+
+.. math::
+    \Upsilon_{\left\{ 1/2, 1/2 \right\}} = (2I +1) \cdot (2S +1) = 4
+
+energy levels and
+
+.. math::
+  \mathcal{N}_{\left\{ 1/2,1/2 \right\}} =
+  \frac{[(2I +1) \cdot (2S +1)]!}{((2I +1) \cdot (2S +1)-2)!}
+  = \frac{[2 \cdot 2]!}{(2 \cdot 0)!} = 12
+
+possible NMR transitions. We write a transition (coherence) from state :math:`i`
+to :math:`j` using the outer product notation :math:`\ketbra{j}{i}`. In
+**mrsimulator**, all simulations are performed in the high-field limit and
+further assume that all spin-spin couplings are in the weak limit.
```

Comment:
high-field limit and simulations further assume

Reply:
I think it needs a comma after further.  Why does it need "simulations"?